2016/04/04 05:30:41 [error] [system.db.CDbCommand] CDbCommand::fetchColumn() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'cheart.authitem' doesn't exist. The SQL statement executed was: SELECT COUNT(*) FROM AuthItem.
in D:\xampp\htdocs\cheart\protected\modules\rights\components\RInstaller.php (180)
in D:\xampp\htdocs\cheart\protected\modules\rights\controllers\InstallController.php (43)
in D:\xampp\htdocs\cheart\index.php (13)
2016/04/04 05:30:44 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'cheart.authassignment' doesn't exist. The SQL statement executed was: SELECT *
FROM `AuthAssignment`
WHERE userid=:userid.
in D:\xampp\htdocs\cheart\protected\modules\rights\components\RAuthorizer.php (341)
in D:\xampp\htdocs\cheart\protected\modules\rights\components\RWebUser.php (21)
in D:\xampp\htdocs\cheart\protected\modules\user\models\UserLogin.php (57)
2016/04/04 05:30:44 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'cheart.authassignment' doesn't exist. The SQL statement executed was: SELECT *
FROM `AuthAssignment`
WHERE userid=:userid' in D:\xampp\htdocs\yii\framework\db\CDbCommand.php:543
Stack trace:
#0 D:\xampp\htdocs\yii\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 D:\xampp\htdocs\yii\framework\web\auth\CDbAuthManager.php(337): CDbCommand->queryAll()
#2 D:\xampp\htdocs\cheart\protected\modules\rights\components\RAuthorizer.php(341): CDbAuthManager->getAuthAssignments('1')
#3 D:\xampp\htdocs\cheart\protected\modules\rights\components\RWebUser.php(21): RAuthorizer->isSuperuser('1')
#4 D:\xampp\htdocs\yii\framework\web\auth\CWebUser.php(246): RWebUser->afterLogin(false)
#5 D:\xampp\htdocs\cheart\protected\modules\user\models\UserLogin.php(57): CWebUser->login(Object(UserIdentity), 0)
#6 D:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): UserLogin->authenticate('password', Array)
#7 D:\xampp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(UserLogin), 'password')
#8 D:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(UserLogin), NULL)
#9 D:\xampp\htdocs\cheart\protected\modules\user\controllers\LoginController.php(19): CModel->validate()
#10 D:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): LoginController->actionLogin()
#11 D:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 D:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 D:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 D:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#15 D:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user/login')
#16 D:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#17 D:\xampp\htdocs\cheart\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/cheart/index.php?r=user/login
HTTP_REFERER=http://localhost/cheart/index.php?r=user/login
---
2016/04/04 05:31:03 [error] [system.db.CDbCommand] CDbCommand::fetchColumn() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'cheart.authitem' doesn't exist. The SQL statement executed was: SELECT COUNT(*) FROM AuthItem.
in D:\xampp\htdocs\cheart\protected\modules\rights\components\RInstaller.php (180)
in D:\xampp\htdocs\cheart\protected\modules\rights\controllers\InstallController.php (43)
in D:\xampp\htdocs\cheart\index.php (13)
2016/04/05 09:08:04 [error] [exception.CException] exception 'CException' with message 'LoginController and its behaviors do not have a method or closure named "layout".' in D:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 D:\xampp\htdocs\cheartportal\protected\modules\user\controllers\LoginController.php(28): CComponent->__call('layout', Array)
#1 D:\xampp\htdocs\cheartportal\protected\modules\user\controllers\LoginController.php(28): LoginController->layout('login')
#2 D:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): LoginController->actionLogin()
#3 D:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 D:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 D:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user/login')
#8 D:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 D:\xampp\htdocs\cheartportal\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/cheartportal/index.php?r=user/login
---
2016/04/05 09:40:56 [error] [exception.CException] exception 'CException' with message 'LoginController cannot find the requested view "login2".' in D:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 D:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('login2', Array, true)
#1 D:\xampp\htdocs\cheartportal\protected\modules\user\controllers\LoginController.php(40): CController->render('login2', Array)
#2 D:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): LoginController->actionLogin()
#3 D:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 D:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 D:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user/login')
#8 D:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 D:\xampp\htdocs\cheartportal\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/cheartportal/index.php?r=user/login
---
2016/04/05 09:41:14 [error] [exception.CException] exception 'CException' with message 'LoginController cannot find the requested view "user/login2".' in D:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 D:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('user/login2', Array, true)
#1 D:\xampp\htdocs\cheartportal\protected\modules\user\controllers\LoginController.php(40): CController->render('user/login2', Array)
#2 D:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): LoginController->actionLogin()
#3 D:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 D:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 D:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user/login')
#8 D:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 D:\xampp\htdocs\cheartportal\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/cheartportal/index.php?r=user/login
---
